app/Traits/CalendarDateTrait.php

```diff
@@ -51,7 +51,22 @@ public function hasCalendarButNoAccess(): bool
     public function getDate(): string
     {
         $reminder = $this->calendarReminder();
-        return $reminder->year . '-' . $reminder->month . '-' . $reminder->day;
+        $months = $reminder->calendar->months();
+        $count = 0;
+        $monthCount = 1;
+        foreach ($months as $month) {
+            if ($month["type"] == "standard") {
+                $count++;
+            }
+            if ($monthCount == $reminder->month) {
+
+                if ($month["type"] == "intercalary") {
+                    return $reminder->year . '-' . $month["name"] . '-' . $reminder->day;
+                }
+                return $reminder->year . '-' . $count . '-' . $reminder->day;
+            }
+            $monthCount++;
+        }
```

Comment:
In case the calendar is broken and has no months, it's still worth having a fallback return at the end of the function what uses the old syntax :)